person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, name, age
+
```

Comment:
- [ ] As `:id` should only be able to be retrieved so only the `Getter` should be defined on it. For that you need to update it and  refer the example below :point_down: 

- [ ] As  `:name `and `:age` instance variables are the only ones that should be able to be modified that's why they should be defined as `attr_accessors` (here both Getters and Setters are defined). Kindly ensure you update them and correct the typo error on each one(`add the missing columns before each one`). i.e.:
```ruby
   ...
    attr_accessor :name, :age
    attr_reader :id: 
   ...
```